### PR TITLE
Move mocks-server.org to docusaurus v2 + DocSearch v3

### PR DESCRIPTION
**Pull request by javierbrea**
# Pull request motivation(s)

The site https://www.mocks-server.org/ have been migrated to Docusaurus V2.

### CODE REVIEW

File: configs/mocks-server.json [hunk 1]
```diff
@@ -10,29 +10,37 @@
   "stop_urls": [],
   "selectors": {
     "lvl0": {
-      "selector": "//*[contains(@class,'navGroups')]//*[contains(@class,'navListItemActive')]/preceding::h3[1]",
-      "type": "xpath",
+      "selector": ".menu__link--sublist.menu__link--active",
       "global": true,
       "default_value": "Documentation"
     },
```
Comment (shortcuts) [inline configs/mocks-server.json:16]:
```json
"lvl0": {
      "selector": "(//ul[contains(@class,'menu__list')]//a[contains(@class, 'menu__link menu__link--sublist menu__link--active')]/text() | //nav[contains(@class, 'navbar')]//a[contains(@class, 'navbar__link--active')]/text())[last()]",
      "type": "xpath",
      "global": true,
      "default_value": "Documentation"
    }
```

This is the selector we use for Docusaurus v2 (handles either navbar or sidebar active link)

File: configs/mocks-server.json [hunk 2]
```diff
@@ -10,29 +10,37 @@
...
+      "version",
+      "type"
+    ],
+    "attributesToRetrieve": [
+      "hierarchy",
+      "content",
+      "anchor",
+      "url",
+      "url_without_anchor",
+      "type"
```
Comment (shortcuts) [inline configs/mocks-server.json:39]:
```suggestion
      "type",
      "docusaurus_tag"
```

This is a needed facet tag for Docusaurus v2

File: configs/mocks-server.json [hunk 3]
```diff
@@ -10,29 +10,37 @@
...
-    ".hash-link"
-  ],
+  "strip_chars": " .,;:#",
   "custom_settings": {
+    "separatorsToIndex": "_",
     "attributesForFaceting": [
       "language",
-      "version"
+      "version",
+      "type"
```
Comment (shortcuts) [inline configs/mocks-server.json:31]:
```suggestion
      "type",
      "docusaurus_tag"
```

This is a needed facet tag for Docusaurus v2

**Comment by shortcuts** [comment]
Hi @javierbrea,

Thanks for the PR, 
I've submitted a few needed changes in order to make Docusaurus v2 + DocSearch v3 work.

I'll merge the PR once it's added

**Comment by javierbrea** [comment]
Hi @shortcuts,
Thank you very much for your help,
I've already added the suggested changes.

**Pull request merged**